MaxText/tests/train_compile_test.py

```diff
@@ -558,7 +558,8 @@ def test_moe_deepseek_without_device_limit(self):
             "topk_routing_group=-1",
         )
     )
-
+  
+  @pytest.mark.skip(reason="b/415132665: Enable it once scan is supported in training for shorter compiler time")
```

Comment:
Is there a testing gap without this? Or are we still covered by other tests?

Reply:
Thanks for the question! Yes, there will be a risk that breaks Llama4 implementation with other features onboarding. I think we are prioritizing the scan version for performance purpose, and should be able to bring it up shortly.